lib/scss_lint/linter/use_variables_for_properties.rb

```diff
@@ -0,0 +1,18 @@
+module SCSSLint
+  class Linter::UseVariablesForProperties < Linter
+    include LinterRegistry
+
+    def visit_root(_node)
+      @included_properties = Set.new(config['included_properties'])
+      yield
+    end
+
+    def visit_prop(node) 
+      return if node.name.count > 1 || !node.name.first.is_a?(String)
+      return unless @included_properties.include? node.name.first
+      return if node.children.first.is_a?(Sass::Script::Tree::Variable)
```

Comment:
What about properties that are the result of a function call? I'm open to thoughts on this, but I can see the value of making this strict about requiring variables.